content/setupd/description.md

```diff
@@ -0,0 +1,7 @@
+# [`wazo-setupd`](https://github.com/wazo-platform/wazo-setupd)
+
+A micro service to initialize a [Wazo](http://wazo.community) Engine.
```

Comment:
Better to use : https://wazo-platform.org

Reply:
Yep, copy/paste from wazo-setupd README :sweat_smile: